client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-hosting-details-list.tsx

```diff
@@ -25,9 +28,34 @@ export const useUpgradePlanHostingDetailsList = () => {
 			),
 			icon: trendingUp,
 		};
-		hostingDetails.splice( 1, 1 );
-		hostingDetails.unshift( updatedHostingSpeedDetails );
+		newDefaultHostingDetails = newDefaultHostingDetails.filter(
+			( { id } ) => id !== 'increased-speed'
+		);
+		newHostingDetails.push( updatedHostingSpeedDetails );
+	}
+
+	// FID hosting details.
+	if ( fidPercentageDifference > 0 ) {
+		const wordpressFID = Math.round( 100 * wordpressCwvtechReportJson?.goodFID );
+
+		const updatedHostingFasterResponseDetails = {
+			title: translate( 'Faster response' ),
+			description: translate(
+				'%(wordpressFidPercentage)s of sites on WordPress.com respond at least %(sitePercentageDifference)s faster than yours on the first interaction.',
```

Comment:
<img width="337" alt="Screenshot 2024-05-14 at 18 28 37" src="https://github.com/Automattic/wp-calypso/assets/876340/44e5c462-9fd0-4713-bb5e-c396baf83dcd">

Different from the proposed design, I implemented it in the same way as https://github.com/Automattic/wp-calypso/pull/90457 when displaying the LCP information. Does it make sense and seem to have more value? Or do we really want to follow the design?

Reply:
We may as well show it if we can. 🤷🏻‍♀️